ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -78,6 +85,9 @@ public void getSize(
       return;
     }
 
+    // Ensure FrescoHelpers.initialize was called.
+    this.initialize();
```

Comment:
Is it always safe according to the API contract? Just because it works doesn't mean it's right.